### ISSUE TITLE
[BUG] Getting a 400 when running on Classic Mainnet (Works Perfectly for Classic Testnet)

### ISSUE THREAD

**Issue by TheArchitect123** [classic-terra/core#150]
I also need to confirm if this is the correct chain-id.
`static mainnetChainId = 'columbus-5';`

I'm getting a 400 when running the same broadcast request for a transaction, which works perfectly for Classic Testnet.
Basically it looks like Mainnet is saying I don't have sufficient gas to cover the fees, but I have 20 LUNC, and I'm trying to transfer only 5 LUNC.

Here's the response I get. I'm invoking Classic Mainnet via a localhost.
![image](https://user-images.githubusercontent.com/20945125/221488463-6690b680-66cc-4a94-8efc-38b9614482eb.png)

**Comment by TheArchitect123** [comment]
I know I have sufficient coins and there's no issue with my recovery keys, and it's not a signing transaction problem.
The blockchain is basically saying I need to transfer a minimum amount.
 
What is the minimum amount required for a transaction? @edk208

Here's my Test (Mainnet) wallet
https://finder.terrac.dev/classic/address/terra1h066yzxpaantknflvvgx0cf6vnr9u6r580ukfj

**Comment by TheArchitect123** [comment]
If there is a minimum amount for a transfer, it should also be documented @edk208

**Comment by edk208** [comment]
there is no minimum amount for a transfer.  the error says you are providing 0uluna fees, but rather using uusd for fees.. is that correct?

**Comment by TheArchitect123** [comment]
LOL, I think I found the issue. Sorry, I forgot to change the Denom for the Classic Tx on my end. 
Let me try again

**Comment by TheArchitect123** [comment]
Ok yeah I fixed it. All good then. Thanks
